ocaml-lsp-server/test/e2e-new/semantic_hl_helpers.ml

```diff
@@ -52,8 +52,9 @@ let annotate_src_with_tokens ~(legend : SemanticTokensLegend.t)
         "<%s%s-%d>"
         token_types.(!token.type_)
         (if annot_mods then
-         "|" ^ String.concat ~sep:"," (modifiers ~legend:token_mods !token.mods)
-        else "")
+           "|"
+           ^ String.concat ~sep:"," (modifiers ~legend:token_mods !token.mods)
+         else "")
```

Comment:
I feel like the old one was better than the new one here.

Reply:
The `then` branch was 1 indented and aligned with the `if` so I wouldn't say better.

How would you prefer this formatted ?

Reply:
The added indentation causes a line break, which is unfortunate.